src/cleanup.ts

```diff
@@ -36,15 +20,41 @@ async function cleanup() {
     }
 }
 
-function addCachedJfToPath(): boolean {
-    // Get the JFrog CLI path from step state. saveState/getState are methods to pass data between a step, and it's cleanup function.
-    const jfCliPath: string = core.getState(Utils.JF_CLI_PATH_STATE);
-    if (!jfCliPath) {
-        // This means that the JFrog CLI was not installed in the first place, because there was a failure in the installation step.
-        return false;
+/**
+ * Executes post tasks related to build information.
+ *
+ * This function performs several tasks after the main build process:
+ * 1. Checks if auto build publish and job summary are disabled.
+ * 2. Verifies connection to JFrog Artifactory.
+ * 3. Collects and publishes build information if needed.
+ * 4. Generates a job summary if required.
+ */
+async function buildInfoPostTasks() {
+    const disableAutoBuildPublish: boolean = core.getBooleanInput(Utils.AUTO_BUILD_PUBLISH_DISABLE);
+    const disableJobSummary: boolean = core.getBooleanInput(Utils.JOB_SUMMARY_DISABLE);
```

Comment:
More efficient. Let's delete Utils.isJobSummarySupported() in line 53.
```suggestion
    const disableJobSummary: boolean = core.getBooleanInput(Utils.JOB_SUMMARY_DISABLE) || Utils.isJobSummarySupported();
```